source/direct_debit/index.html.md.erb

```diff
@@ -5,285 +5,259 @@ weight: 80
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> ACTIVE </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-  </table>
-</div>
+`return_url` is an HTTPS URL on your site. GOV.UK Pay will direct your user to this URL when GOV.UK Pay have finished setting up the mandate.
 
-## One-off payments using Direct Debit
+The URL should not be JSON-encoded, because backslashes are invalid characters in the API.
 
-To take a single payment using Direct Debit, you must set up a Direct Debit mandate.
+#### description (optional)
 
-1. Create an API key linked to your Direct Debit service.
+`description` is a human-readable description of the mandate.
 
-1. Follow the instructions in the [“Payment flow” section](/payment_flow) of this documentation.
+The description must be no longer than 255 characters, and must not contain URLs.
 
-The same API call is used for single payments in all contexts, including
-Direct Debit:
+### 2. Direct the user to GOV.UK Pay
 
-POST /v1/payments
+When the API responds, you need to:
 
+- direct the user to the `next_url` from the API response
+- store the `mandate_id` from the API response, so you can use that ID to collect payments against the mandate later
 
-You cannot take further payments from users using this method without creating
-a new payment. This requires users to re-enter their account details.
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-## Variable payments using Direct Debit
+### 3. The user provides their bank account details
 
-You can collect payments of variable amounts at any time by using a variable agreement.
-For example you may want to collect:
+When your service redirects the user to `next_url`, the user will see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-* money each time a service is provided
+GOV.UK Pay checks with GoCardless that the user’s account number and sort code match a real bank account.
 
-* payments where the amounts vary according to service usage
+### 4. GOV.UK Pay presents the user with a confirmation screen
 
-* regular payments where you do not know in advance how much each payment will be
+The screen presents the user’s account details. The user grants their consent
+by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
+that their mandate has been set up. If the mandate then fails, the
+GOV.UK Pay team will send an email to update the user that their mandate could
+not be set up.
```

Comment:
“GOV.UK Pay sends an email to the user to confirm that their mandate has been set up” makes it sound a bit too much like this can’t fail. We should says something along the lines of “GOV.UK Pay sends an email to the user to inform them that their mandate is being set up”.

Chronologically, the “If the mandate then fails, the GOV.UK Pay team will send an email to update the user that their mandate could not be set up” bit happens after we’ve sent the details to GoCardless, so maybe that should be moved into the next step.

Reply:
Thanks Alex - will amend!